packages/react-tc-ui-kit/src/assets/images/eye-slash.svg

```diff
@@ -0,0 +1 @@
+<svg width="25" height="20" viewBox="0 0 25 20" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M12.17 15.63a5.52 5.52 0 01-5.43-5.2l-4-3.17c-.52.67-1 1.39-1.39 2.17a1.3 1.3 0 000 1.14 12.18 12.18 0 0010.82 6.93c1.03 0 2.02-.16 2.97-.4l-1.98-1.58c-.32.07-.65.1-.99.1zm11.94 2.26l-4.2-3.33A12.89 12.89 0 0023 10.57a1.29 1.29 0 000-1.14A12.18 12.18 0 0012.17 2.5c-1.95 0-3.88.5-5.6 1.47L1.73.13a.6.6 0 00-.68-.05.61.61 0 00-.18.16l-.74.99a.64.64 0 00.1.88l22.39 17.76a.6.6 0 00.68.05c.06-.04.13-.1.17-.16l.75-.99a.64.64 0 00-.1-.88zm-6.99-5.54l-1.5-1.19A3.8 3.8 0 0014.4 7a3.56 3.56 0 00-3.19-.6c.23.32.36.7.36 1.1l-.06.4-2.8-2.23a5.33 5.33 0 017.34.35 5.63 5.63 0 011.07 6.33z" fill="#172F3D"/></svg>
```

Comment:
pls remove `width` and `height` attrs\
`fill` attr on path should be `currentColor`, not hardcoded color value\
then we should add className for svgs and css should be `display: inline-block; width: 1.25em; height: 1em;`